src/request/components/request-detail-panel-logging.tsx

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import { ILogMessage } from '../messages/ILogMessage';
+import { ILoggingViewModel } from '../view-models/ILoggingViewModel';
+
+import React = require('react');
+
+interface ILogMessagesProps {
+    messages: ILogMessage[];
+}
+
+/**
+ * React class to display console messages
+ */
+class LogMessages extends React.Component<ILogMessagesProps, {}> {
```

Comment:
Now that this is simplified, I wonder if there is any benefit to having this in a separate component?